.env

```diff
@@ -1,17 +1,18 @@
 APP_DB_URL="jdbc:postgresql://127.0.0.1:5433/jpc?reWriteBatchedInserts=true&user=jpc&password=letmein"
-BASM_BUCKET_NAME=basm
-JPC_BUCKET_NAME=jpc
 AWS_ACCESS_KEY_ID=foo
 AWS_SECRET_ACCESS_KEY=bar
 AWS_DEFAULT_REGION=eu-west-2
-S3_ENDPOINT=http://localhost:4572
-IMPORT_FILES_LOCATIONS=schedule_34_locations_sample.xlsx
-IMPORT_FILES_PRICES_SERCO=serco_sample.xlsx
-IMPORT_FILES_PRICES_GEO=geoamey_sample.xlsx
+BASM_API_BASE_URL=http://localhost:9999
```

Comment:
This is the new addition.  This is pointing at WireMock that starts up via the docker-compose (done in a separate PR).